hack/util.sh

```diff
@@ -74,7 +74,7 @@ function configure_os_server() {
 	openshift start \
 	--write-config=${SERVER_CONFIG_DIR} \
 	--create-certs=false \
-	--dns="tcp://${API_HOST}:53" \
+	--dns="tcp://${API_HOST}:8053" \
```

Comment:
I will remove that before the final commit - it's just something to fix a weird issue my local env was having,